docs/rules/no-this-render.md

```diff
@@ -0,0 +1,31 @@
+# no-this-render
+
+Ember's `this.render` method and [`@ember/test-helpers`](https://github.com/emberjs/ember-test-helpers)'s `render` method are equivalent, but using `test-helpers`' `render` method is the recommended approach.
```

Comment:
Instead of saying `test-helpers` can you say `@ember/test-helpers` just to avoid any confusion / ambiguity?